components/financial_assistance/app/views/financial_assistance/applications/review.html.erb

```diff
@@ -1,344 +1,450 @@
+      <dt class="mr-4"><%= l10n('homepage.applications.detail.id') %></dt>
+      <dd><%= @application.hbx_id %></dd>
+    </dl>
+
+    <%# Income and Deductions table %>
+    <% if income_and_deductions_for_any(@application).present? %>
+      <section>
+        <h2><%= l10n("faa.deductions") %></h2>
+        <div>
+          <div><%= l10n("faa.review.income.information_entered") %></div>
+          <table>
             <thead>
-              <tr class="form-heading">
-                <th class="no-bottom-border"> <b>PERSON </b></th>
-                <th class="no-bottom-border"><b>TYPES</b></th>
-                <th class="no-bottom-border"><b>FREQUENCY</b></th>
-                <th class="no-bottom-border"><b>AMOUNT</b></th>
-                <th class="no-bottom-border"><b>DATES </b></th>
+              <tr>
+                <th><%= l10n("person") %></th>
+                <th><%= l10n("faa.review.income.table.types") %></th>
+                <th><%= l10n("faa.review.income.table.frequency") %></th>
+                <th><%= l10n("faa.review.income.table.amount") %></th>
+                <th><%= l10n("faa.review.income.table.dates") %></th>
               </tr>
             </thead>
             <tbody>
               <% @applicants.each do |applicant| %>
                 <% income_and_deductions_for(applicant).each do |embedded_document| %>
                   <tr>
                     <td> <%= applicant.full_name %> </td>
-                    <td> <%= l10n("faa.#{embedded_document.class.name.demodulize.downcase}.#{embedded_document.kind}") %> <%= FinancialAssistance::Income::SS_BENEFITS[embedded_document.ssi_type&.to_sym] if  embedded_document.kind == "social_security_benefit" && FinancialAssistanceRegistry.feature_enabled?(:ssi_income_types) %></td>
+                    <td> <%= l10n("faa.#{embedded_document.class.name.demodulize.downcase}.#{embedded_document.kind}") %> <%= FinancialAssistance::Income::SS_BENEFITS[embedded_document.ssi_type&.to_sym] if embedded_document.kind == "social_security_benefit" && FinancialAssistanceRegistry.feature_enabled?(:ssi_income_types) %></td>
                     <td> <%= embedded_document.frequency_kind.humanize %> </td>
                     <td>
-                    <b>
-                      <% if embedded_document.is_a? FinancialAssistance::Deduction %>
-                        (<%= number_to_currency(embedded_document.amount.to_s) %>)
-                      <% else %>
-                        <%= number_to_currency(embedded_document.amount.to_s) %>
-                      <% end %>
-                    </b>
+                      <% amount = number_to_currency(embedded_document.amount.to_s) %>
+                      <%= embedded_document.is_a?(FinancialAssistance::Deduction) ? "(#{amount})" : amount %>
                     </td>
                     <td>
-                      <%= start_to_end_dates(embedded_document) %>
+                      <div class="d-flex justify-content-between">
```

Comment:
nitpick- here and on ln26 the extra div is likely unnecessary. Any classes can just be applied to the parent